<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>767</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainDialog</string>
  </property>
  <widget class="QTreeView" name="treeView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>431</width>
     <height>641</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pBLoad">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>690</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load XML File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBSave">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>690</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save XML File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_ReadDirStruct">
   <property name="geometry">
    <rect>
     <x>344</x>
     <y>690</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Read Dir Struct</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>730</y>
     <width>431</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="MyLineEditClass" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>230</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>401</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>210</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Base Project</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pMKSGenButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>270</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Create Prj in MKS</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>27</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>40</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>MKS Server</string>
   </property>
  </widget>
  <widget class="MyLineEditClass" name="mks_serveredit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>60</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>90</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>MKS Port</string>
   </property>
  </widget>
  <widget class="MyLineEditClass" name="mks_portedit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>110</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>140</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Change Package</string>
   </property>
  </widget>
  <widget class="MyLineEditClass" name="changePackageEdit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>160</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyLineEditClass</class>
   <extends>QLineEdit</extends>
   <header>mylineeditclass.h</header>
   <slots>
    <slot>mousePressEvent(QMouseEvent)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>treeView</tabstop>
  <tabstop>pBLoad</tabstop>
  <tabstop>pBSave</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
